[core] Batch small changes (#17910)
mui-org/material-ui: docs/src/pages/components/snackbars/snackbars.md

![stars](https://img.shields.io/github/stars/iamhosseindhv/notistack.svg?style=social&label=Stars)
![npm downloads](https://img.shields.io/npm/dm/notistack.svg)

> We demonstrate how to use [notistack](https://github.com/iamhosseindhv/notistack).

Reviewer: ```suggestion
This example demonstrates how to use [notistack](https://github.com/iamhosseindhv/notistack).
```

> notistack has an **imperative API** that makes it easy to display toasts (so you don't have to deal with open/close state of them).

Reviewer: ```suggestion
notistack has an **imperative API** that makes it easy to display snackbars, without having to handle their open/close state.
```

It also enables you to **stack** them on top of one another (although this is discouraged by the Material Design specification).

> 

Reviewer: ```suggestion
It also enables you to stack them on top of one another (although this is discouraged by the Material Design specification).
```
